### PR TITLE
BREAKING CHANGE: Support tron signature verification

### PR DESCRIPTION
**Pull request by zeroqn**
Add Secp256k1Tron to account lock manage
## Breaking Change
Add `tron_account_lock` to `programs` in scripts-deploy.json

### CODE REVIEW

File: crates/block-producer/src/runner.rs [hunk 1]
```diff
@@ -136,6 +139,10 @@ pub fn run(config: Config) -> Result<()> {
             eth_lock_script_type_hash.unpack(),
             Box::new(Secp256k1Eth::default()),
         );
+        account_lock_manage.register_lock_algorithm(
+            eth_lock_script_type_hash.unpack(),
```
Comment (jjyr) [inline crates/block-producer/src/runner.rs:143]: We do not enable this lock in the current release, Pls remove this

File: crates/tools/src/deploy_genesis.rs [hunk 2]
```diff
@@ -201,9 +201,10 @@ pub fn deploy_genesis(
         GwPack::pack(&deployment_result.polyjuice_validator.script_type_hash),
     ];
 
-    let mut allowed_eoa_type_hashes: Vec<gw_packed::Byte32> = vec![GwPack::pack(
-        &deployment_result.eth_account_lock.script_type_hash,
-    )];
+    let mut allowed_eoa_type_hashes: Vec<gw_packed::Byte32> = vec![
+        GwPack::pack(&deployment_result.eth_account_lock.script_type_hash),
+        GwPack::pack(&deployment_result.tron_account_lock.script_type_hash),
```
Comment (jjyr) [inline crates/tools/src/deploy_genesis.rs:206]: Remove this line

**Review by jjyr — CHANGES_REQUESTED** [review]

File: crates/block-producer/src/runner.rs (hunk 1 above)
Comment (zeroqn) [reply crates/block-producer/src/runner.rs]: Done

File: crates/tools/src/deploy_genesis.rs (hunk 2 above)
Comment (zeroqn) [reply crates/tools/src/deploy_genesis.rs]: Done

**Review by jjyr — APPROVED** [review]

**Pull request merged**